installer/conf/kube.conf

```diff
@@ -55,7 +55,7 @@
 #Kubernetes health
 <source>
   type kubehealth
- tag oms.api.KubeHealth.ReplicaSet
+ tag kubehealth.ReplicaSet
```

Comment:
kube.conf changes requires change in omsdocker/.yaml (for aks-engine) and the chart (for AKS). Please ensure you change both as part of roll out.

Reply:
Yes. that is the plan